src/admin_extra/service_admin_extra_gdpr.erl

```diff
@@ -144,6 +144,13 @@ get_logs(Username, Domain, TmpDir) ->
 get_logs_from_node(Node, Username, Domain, TmpDir) ->
     {ok, ZippedData} = rpc:call(Node, ?MODULE, retrieve_logs, [Username, Domain]),
     {ok, [File]} = zip:unzip(ZippedData, [{cwd, TmpDir}]),
-    string:prefix(File, TmpDir ++ "/").
-
+    prefix(File, TmpDir ++ "/").
 
+%% for OTP version < 20
+-ifdef(OTP_RELEASE).
```

Comment:
how do we check exactly that the version is <20 ?

Reply:
We don't, both implementations should work above OTP 20, but only the `else` part below the OTP 20 version. `OTP_RELEASE` macro was introduced in OTP 21 therefore we use `string:prefix/2` for versions >= 21 and list subtraction otherwise. This can be removed when we stop supporting OTP 19.X.

Reply:
Similar idea is used in `gen_fsm` http://erlang.org/doc/man/gen_fsm.html .